server/api.go

```diff
@@ -927,7 +927,9 @@ func (a *API) siteStats(w http.ResponseWriter, r *http.Request) {
 		return
 	}
 
-	stats, err := a.p.store.GetStats(a.p.GetRemoteID(), a.p.GetPreferenceCategoryName())
+	stats, err := a.p.store.GetStats(storemodels.GetStatsOptions{
+		Stats: []storemodels.StatType{storemodels.StatsConnectedUsers},
```

Comment:
I realized that we are adding more stats to parse from `GetStats()` but this API call just needs one of them, so I added a way to only parse certain stats when needed

Reply:
Commented below, but wondering if it would be simpler just to expose discrete store methods instead?